collectors/charts.d.plugin/nvme/README.md

```diff
@@ -0,0 +1,65 @@
+<!--
+title: "Storage NVMe devices monitoring with Netdata"
+custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/charts.d.plugin/nvme/README.md
+-->
+
+# Storage NVMe devices monitoring with Netdata
+
+Collect information from NVMe devices.
+
+Executed commands:
+
+- `sudo -n nvme list`
+
+
+## Requirements
+
+- Install: nvme-cli
+
+The module uses `nvme-cli`, which can only be executed by `root`. It uses
+`sudo` and assumes that it is configured such that the `netdata` user can execute `nvme` as root without a password.
+
+- Add to your `/etc/sudoers` file:
+
+`which nvme` shows the full path to the binary.
+
+```bash
+netdata ALL=(root)       NOPASSWD: /path/to/nvme
+```
+
+- Reset Netdata's systemd
+  unit [CapabilityBoundingSet](https://www.freedesktop.org/software/systemd/man/systemd.exec.html#Capabilities) (Linux
+  distributions with systemd)
+
+The default CapabilityBoundingSet doesn't allow using `sudo`, and is quite strict in general. Resetting is not optimal, but a next-best solution given the inability to execute `nvme` using `sudo`.
+
+
+As the `root` user, do the following:
+
+```cmd
+mkdir /etc/systemd/system/netdata.service.d
+echo -e '[Service]\nCapabilityBoundingSet=~' | tee /etc/systemd/system/netdata.service.d/unset-capability-bounding-set.conf
+systemctl daemon-reload
+systemctl restart netdata.service
+```
+
+
+## Charts
+- Critical Warning
+- Percentage Used
+- Temperature
+- Power cycles
+- Power on hours
```

Comment:
```suggestion
- Critical Warning
- Percentage Used
- Temperature
- Power cycles
- Power on hours
```
@aazedo 
Could you add a description for each chart? 
This page gives you an example of chart descriptions: https://learn.netdata.cloud/docs/agent/collectors/python.d.plugin/beanstalk